src/vivarium_csu_ltbi/components/treatment.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+
+
+# noinspection PyAttributeOutsideInit
+class BaselineCoverage:
+
+    @property
+    def name(self):
+        return 'treatment_coverage'
+
+    def setup(self, builder):
+        self.treatment_stream = builder.randomness.get_stream('treatment_propensity')
+
+        self.household_tb_exposure = builder.value.get_value('household_tuberculosis.exposure')
+
+        # Setup pipelines for 6H coverage for the two treatment subgroups
+        six_h_coverage_data = builder.data.load("six_h.coverage.proportion")
+        six_h_with_hiv, six_h_under_five_hhtb = self.setup_coverage_tables(builder, six_h_coverage_data)
+        self.six_h_coverage_with_hiv = builder.value.register_value_producer('six_h_with_hiv',
+                                                                             source=six_h_with_hiv)
+        self.six_h_coverage_under_five_hhtb = builder.value.register_value_producer('six_h_under_five_hhtb',
+                                                                                    source=six_h_under_five_hhtb)
+
+        # Setup pipelines for 3HP coverage for the two treatment groups
+        three_hp_coverage_data = builder.data.load("three_hp.coverage.proportion")
+        three_hp_with_hiv, three_hp_under_five_hhtb = self.setup_coverage_tables(builder, three_hp_coverage_data)
+        self.three_hp_coverage_with_hiv = builder.value.register_value_producer('three_hp_with_hiv',
```

Comment:
These pipelines require the age and sex columns

Reply:
Also name them in the same style as other pipelines:

three_hp_with_hiv.coverage, etc.